src/index.js

```diff
@@ -1,4 +1,4 @@
-import grapesjs from 'grapesjs';
+//import grapesjs from 'grapesjs';
```

Comment:
@Bakino can you explain why this was commented out? It seems this is the problem with issue #41 . I removed the comment and published another version but I would like to know if there are any side effects.